### PR TITLE
Upgrade required node version to 8.x and npm to 5.x for builder-web

### LINKED ISSUE #2986: Update Node version for builder-web
Current version of node-sass is only compatible with Node v4.x.x. Node is currently at v8.x.x. Will also require updating most of the npm packages and running full regression to be sure this doesn't break anything.

### PR DESCRIPTION
**Pull request by elliott-davis**
Closes #2986

tenor-27785143

Signed-off-by: Travis Elliott Davis <edavis@chef.io>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: components/builder-web/.fnmrc [hunk 1]
```diff
@@ -1 +1 @@
-4.4.3
+8.7.0
```
Comment (raskchanky) [inline components/builder-web/.fnmrc]: This file can be deleted.  I figured out a better way to run nvm from fish.

**Comment by raskchanky** [comment]
Serious non-trolling question:  any thoughts on using yarn?

**Comment by elliott-davis** [comment]
@raskchanky now that npm5 is out it has most if not all the benefits of yarn

**Comment by reset** [comment]
tenor-25773964

**Pull request merged**